### PR TITLE
Charts: update the 'Dynamic Series from the DataSource' demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/SeriesTemplates/description.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    [
+        {seriesName: series1, arg: arg11Value, val: value11 }
+        {seriesName: series1, arg: arg12Value, val: value12 }
+        ...
+        {seriesName: seriesM, arg: argM1Value, val: valueM1 }
+        {seriesName: seriesM, arg: argM2Value, val: valueM2 }
+        ...
+    ]
+
+Each object in the data source must represent a point in a single series.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/SeriesTemplates/description.md]: Never use the word "represent". **Under no circumstances this word can be in our documentation**. This is utterly important. Please refer to the following guideline topic for details: https://tw.devexpress.devx/language-guidelines/misused-words/represent/

```suggestion
Every object in the data source should correspond to a point in a single series.
```

File: JSDemos/Demos/Charts/SeriesTemplates/description.md [hunk 2]
```diff
@@ -1 +1,26 @@
-In&nbsp;this example, data for series is&nbsp;provided in&nbsp;a&nbsp;peculiar manner. Each object in&nbsp;the data source represents a&nbsp;point in&nbsp;a&nbsp;single series. To&nbsp;define series in&nbsp;this case, use the **seriesTemplate** object where you need to&nbsp;specify the data field providing series names and custom settings for a&nbsp;given series. Settings common to&nbsp;all series must be&nbsp;set within the **commonSeriesSettings** object. Providing data in&nbsp;this manner is&nbsp;useful when the number of&nbsp;points within a&nbsp;single series changes over time.
\ No newline at end of file
+In some scenarios, you need to add more series to the data source after you created the Chart. In this case, organize your data source structure as follows:
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/SeriesTemplates/description.md]:
```suggestion
In certain scenarios, you may need to add more series to the data source after you created the Chart. In this case, arrange your data source structure as follows:
```

File: JSDemos/Demos/Charts/SeriesTemplates/description.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+        {seriesName: series1, arg: arg12Value, val: value12 }
+        ...
+        {seriesName: seriesM, arg: argM1Value, val: valueM1 }
+        {seriesName: seriesM, arg: argM2Value, val: valueM2 }
+        ...
+    ]
+
+Each object in the data source must represent a point in a single series.
+
+This demo uses the structure displayed above to organize the data:
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/SeriesTemplates/description.md]: The noun "data" is uncountable, therefore, you should not use an article with it unless you have a very strong reason to do so. In this case, the definite article places extra emphasis on data. I do not feel that we need it here.

```suggestion
This demo uses the structure displayed above to organize data:
```

File: JSDemos/Demos/Charts/SeriesTemplates/description.md [hunk 4]
```diff
@@ -1 +1,26 @@
...
+
+    {
+        year: 1970,
+        country: 'Saudi Arabia',
+        oil: 192.2,
+    }
+
+To define series, use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify common settings for all series: the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField), the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#valueField) and the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#type).
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/SeriesTemplates/description.md]: Mind the Oxford comma: https://tw.devexpress.devx/punctuation/#use-comma

```suggestion
To define series, use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object to specify common settings for all series: the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField), the [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#valueField), and the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#type).
```

File: JSDemos/Demos/Charts/SeriesTemplates/description.md [hunk 5]
```diff
@@ -1 +1,26 @@
...
+
+Then, use the [seriesTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/) configuration object to define a template for the series. Within this object, assign the data source field that specifies the series name to the [nameField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#nameField) property.
+
+If you need to specify individual values for properties of a particular series, assign a callback function to the [customizeSeries](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#customizeSeries) property of the [seriesTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/) object. This demo uses the [customizeSeries](/Documentation/ApiReference/UI_Components/dxChart/Configuration/seriesTemplate/#customizeSeries) function to display a line instead of a bar for the `year: 2009`.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/SeriesTemplates/description.md]: Is this an expression? `year: 2009`

If it is, then you do not need an article.

Comment (vladaskorohodova) [reply JSDemos/Demos/Charts/SeriesTemplates/description.md]: Yes, it is an expression. Noted

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**